README.md

```diff
@@ -28,3 +59,24 @@ Example: (Updated)
 More details on experiment-buddy:
 1. experiment-buddy will add tagged commits to a dangling branch 
 2. Supports: Unix based OS
+
+## For the tester
+
+#### Local flow
+```shell
+docker run -v ~/.ssh:/root/.ssh --rm -ti \
+           -e WANDB_API_KEY=$WANDB_API_KEY \
+           -e GIT_MAIL=$(git config user.email) \
+           -e GIT_NAME=$(git config user.name) \
+           -u root:$(id -u $USER) $(docker build -f ./Dockerfile-flow -q .)
```

Comment:
`docker build -q` is too quite here. If the purpose is to test, I think verbose is more user friendly.

Reply:
Yes and no, `Dockerfile-flow` is a bare-bone python image with just the needed packages preinstalled to speed up the pip installation, nothing else, the whole test is made during the run of the container, there you have all the needed logs. This command overcomes the absence of a `--build` option during the run and reduce the startup routine to a single copy&paste

A possible alternative would be to split the build and the run in two separate commands and remove the `-q`, but, given that the build output has no purpose other than telling that it has downloaded a python image, I'd prefer to keep the noise as low as possible